submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestIndexer.kt

```diff
@@ -22,10 +22,8 @@ class SubmissionRequestIndexer(
         val sub = request.submission
 
         logger.info { "${sub.accNo} ${sub.owner} Started indexing submission files" }
-
         val totalFiles = indexSubmissionFiles(sub)
-        requestService.updateRequestTotalFiles(accNo, version, totalFiles)
-        requestService.updateRequestStatus(accNo, version, INDEXED)
+        requestService.saveSubmissionRequest(request.withNewStatus(INDEXED, totalFiles))
```

Comment:
rather that perform two individual updates status and files we can simple call save operation,